.github/workflows/npa.yml

```diff
@@ -13,8 +13,8 @@ on:
 
 env:
   GO_VERSION: "1.22.0"
-  K8S_VERSION: "v1.30.0"
-  KIND_VERSION: "v0.23.0"
+  K8S_VERSION: "v1.30.4"
+  KIND_VERSION: "v0.24.0"
```

Comment:
ok, so is the kind version that is the first one with network policies embedded ... it seems there is some bad interactions